docs/podcasts/35.md

```diff
@@ -0,0 +1,14 @@
+---
+  template: podcast.html
+  title: "Episode 35: The Perfect Media Server"
+  number: 35
```

Comment:
whats the number for? as its already in the title?

Reply:
This is used to create the link to the episode and the chapter links.

Reply:
hmm ok then that makes sense